docs/benchmarks/micro-benchmarks.md

```diff
@@ -80,34 +68,12 @@ id: micro-benchmarks
         <ul><li><b>Storage Benchmark</b></li>
           <ul><li><b>Disk</b></li>
             <ul>
-              <li>Read/Write</li><li>Rand_Read/Rand_Write</li>
+              <li>Seq Read/Seq Write</li><li>Rand_Read/Rand_Write</li>
```

Comment:
Better align with other naming. For example:
Seq_Read/Seq_Write
Rand_Read/Rand_Write
Seq_R/W_Read
Seq_R/W_Write
Rand_R/W_Read
Rand_R/W_Write